source/cloud/aws/eks.md

```diff
@@ -20,24 +20,32 @@ $ aws configure
 
 ## Create the Kubernetes cluster
 
-Now we can launch a GPU enabled EKS cluster. First launch an EKS cluster with `eksctl`.
+Now we can launch a GPU enabled EKS cluster with `eksctl`.
+
+```{note}
+1. You will need to create or import public ssh-key to be able to execute the following command.
```

Comment:
```suggestion
1. You will need to create or import a public SSH key to be able to execute the following command.
```

I think the spelling `ssh-key` makes it seem like this might be a command, when really it's referring to just a keypair in general. Would you consider this rewording?